### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by obj-obj**
Tune formatting to match our formatting style and fix other minor issues

### CODE REVIEW

**Comment by obj-obj** [comment]
Should probably be merged without the last commit to avoid merge conflicts

File: src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java:165]: Please join these lines.

Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java:165]: Please join these lines.

File: eclipse-formatter.xml
Comment (jamesdooley4) [inline eclipse-formatter.xml:284]: I'd recommend leaving this at 'false' so lines like these don't get squished all together and become harder to read. Completely fine with joining lines in comments:
```
   public SwerveControllerCommand(
            Trajectory trajectory,
            Supplier<Pose2d> pose,
            SwerveDriveKinematics kinematics,
            PIDController xController,
            PIDController yController,
```

Comment (jamesdooley4) [reply eclipse-formatter.xml:284]: And to reformat things without a bunch of manual work, just go back to a previous commit (e.g. the current main branch HEAD) with `git checkout <commitID> .`, then unstage the files and re-apply the new formatting.

Comment (obj-obj) [reply eclipse-formatter.xml:284]: I can also just use `git reset <hash>`, and yeah I'll do that

**Review by bobtseattle — COMMENTED** [review]
There seem to be some inconsistent formatting of parameters that may have gotten us on a bit of a formatting treadmill.
There are many conflicting examples of one param per line vs as many as can fit per line.  We run the risk of them flapping back and forth depending on who was in the code last.

Regarding command sequences, my opinion is that placing multiple commands from a command sequence on a single line hurts readability.

File: src/main/java/frc/team2412/robot/sim/PhysicsSim.java
Comment (obj-obj) [reply src/main/java/frc/team2412/robot/sim/PhysicsSim.java:85]: I think it only splits lines if the parameters are beyond a certain length or something like that

File: eclipse-formatter.xml
Comment (obj-obj) [reply eclipse-formatter.xml:284]: I think after we tune the formatter enough to match our coding style, it won't be needed

**Comment by obj-obj** [comment]
I don't think there are any other issues after looking through stuff

File: eclipse-formatter.xml
Comment (obj-obj) [reply eclipse-formatter.xml:284]: Current level of tuning _should_ be enough if there aren't any other issues

File: src/main/java/frc/team2412/robot/Robot.java
Comment (obj-obj) [inline src/main/java/frc/team2412/robot/Robot.java:90]: This is questionable, the one thing that'd be left to do is fix this (unless other people think it's fine)

Comment (TAKBS2412) [reply src/main/java/frc/team2412/robot/Robot.java:90]: If there's a simple way to fix this without reformatting other parts I'd suggest fixing it, I can imagine this being a bit troublesome down the line.

Comment (obj-obj) [reply src/main/java/frc/team2412/robot/Robot.java:90]: There is a way to fix it but that would disable formatting for all other array initializers. There's like zero documentation on eclipse's xml file so all I can really do is tweak the value until it works

Comment (obj-obj) [reply src/main/java/frc/team2412/robot/Robot.java:90]: I'd say leave it because we want to keep areas like DrivebaseSubsystem readable

**Review by jamesdooley4 — CHANGES_REQUESTED** [review]
Per Teams thread, please hold off on this until after the Sammamish competition. Probably best to do this right after we merge back in the competition change.

File: eclipse-formatter.xml
Comment (jamesdooley4) [inline eclipse-formatter.xml:17]: @obj-obj  - these "magic" values are bitflags defined here: https://www.cct.lsu.edu/~rguidry/ecl31docs/api/constant-values.html#org.eclipse.jdt.internal.formatter.align.Alignment.M_INDENT_ON_COLUMN
See `createAlignmentValue` in https://www.cct.lsu.edu/~rguidry/ecl31docs/api/src-html/org/eclipse/jdt/core/formatter/DefaultCodeFormatterConstants.html for which constants are valid

Comment (obj-obj) [reply eclipse-formatter.xml:17]: Ayyy actual documentation?

**Comment by obj-obj** [comment]
Ideally I'd be able to fully disable line wrapping done by the formatter, while keeping arguments on seperate lines - currently it's tuned to look fine in my VSCode, but depending on other people's font sizes and sidebar width, it might look bad on other configurations - which is why it's best for the editor to handle line wrapping

**Comment by obj-obj** [comment]
Above commit can be left out, the reasoning for using tabs is that the tab size can be configured per-editor, allowing everyone to set their indentation level to whatever they want without editing the file

**Comment by obj-obj** [comment]
Actually it would be better as an issue, ill force push it out

**Comment by obj-obj** [comment]
Should I revert indentation to 4 spaces?

**Pull request closed without merging**
